### DISCUSSION TITLE
ne pas observer une descente de la température intérieure hors occupation ne veut pas dire qu'on économise pas d'énergie

### DISCUSSION THREAD

**Discussion by alexandrecuer** [Open-Building-Management/EnergyGym#9] [la performance énergétique des bâtiments]
# 1604536340
![1](https://user-images.githubusercontent.com/24553739/204369032-dc30344c-8260-4641-861f-b1db6108d7dd.png)
![1_mirror](https://user-images.githubusercontent.com/24553739/204369042-3bb6c847-0af4-4e75-9c39-f5e4b29f4e71.png)
# 1646753540
![2](https://user-images.githubusercontent.com/24553739/204369050-5ef13785-2d1c-4b4d-97a9-a452e1094c3d.png)
![2_mirror](https://user-images.githubusercontent.com/24553739/204369675-720d1ca7-590d-497e-9b04-06cfbd14a592.png)
# 1615991540
![3](https://user-images.githubusercontent.com/24553739/204369076-8ae87d33-d061-4860-8cdf-58809f14f44d.png)
![3_mirror](https://user-images.githubusercontent.com/24553739/204369091-79b2a12a-eea7-42cf-ab72-4bee2bc36fd5.png)
# 1653708740
![4](https://user-images.githubusercontent.com/24553739/204369099-b78b930a-ce1a-4169-ada4-31c641b811c1.png)
![4_mirror](https://user-images.githubusercontent.com/24553739/204369110-6787594f-0dc7-4506-b9f8-28d228494afe.png)
# 1615915940
![5](https://user-images.githubusercontent.com/24553739/204369119-825b90c5-f890-4bf8-9d8c-88f61ebfc085.png)
![5_mirror](https://user-images.githubusercontent.com/24553739/204369138-054ccddf-06a1-48c9-8888-fa661c42b7a7.png)

**Comment by AndreiMkRosa** [comment]
Bonjour. Uniquement pour être sûr, sont-ces graphiques générés par le mode Vacancy?

**Comment by alexandrecuer** [comment]
@AndreiMkRosa : oui les graphiques sont produits en mode vacancy
Pour chaque timestamp, le premier graphique représente le jeu du réseau, le second est obtenu en calculant combien le réseau a économisé d'énergie pendant son jeu. S'il a économisé x heures, alors, on arrête de chauffer pendant x heures, puis on chauffe jusqu'à l'arrivée du personnel.
Je n'ai pas trouvé la récompense à attribuer pour que le réseau converge vers une politique du type du second graphique. Mais ce n'est pas très important je pense. 
A noter que les chauffagistes et les thermiciens du bâtiment ont tendance à critiquer le premier graphique dans le sens où il use plus le matériel : c'est vrai pour la pompe, mais c'est faux pour la vanne : en mode loi d'eau, une vanne est sollicitée très souvent vu qu'elle régule la température du fluide....

**Reply by alexandrecuer** [reply]
@AndreiMkRosa : pour votre information, j'ai clarifié l'écriture de la récompense en mode vacancy
c'était un oubli de ma part d'avoir laissé cette écriture alors que la chose est plutôt simple en fait 
https://github.com/Open-Building-Management/EnergyGym/blob/0c68aa78eff77c15c1fb7bf196f140457205f951/energy_gym/heatgym.py#L219
si celà se trouve, sans mutiplier par -15 (seulement par -1), celà marche exactement pareil. il faudrait essayer
